app/design/frontend/base/default/template/oyst/oneclick/checkout/cart/oneclick.phtml

```diff
@@ -23,10 +23,12 @@
 <script>
     (function () {
         if ("function" === typeof oystOneClick) {
-            oystOneClick(
-                <?php echo $this->getOneClickUrl(); ?>,
-                <?php echo Mage::getSingleton('checkout/session')->getQuote()->getId(); ?>
-            );
+            var oystOneClickConfig = {
+                'oneClickUrl': <?php echo $this->getOneClickUrl(); ?>,
+                'isCheckoutCart': true,
```

Comment:
Checks JavaScript source code for last comma in object literal. Such object could have problem in certain JavaScript engine implementations.